lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/AllocatedEvaluator.java

```diff
@@ -65,9 +65,17 @@
    *
    * @param type
    */
+  @Deprecated
```

Comment:
This is purely internal code. Hence, I'd be OK with just dropping it straight without keeping the deprecated code around for a release.